week3/project/hackyourweather/src/components/CityCardList.js

```diff
@@ -0,0 +1,30 @@
+import CityCard from "./CityCard";
+import "./CityCardList.css";
+
+const CityCardList = ({ cityWeather, removeCity }) => {
+  return (
+    <div className="flex-container">
+      {cityWeather.map(weatherData => {
```

Comment:
```suggestion
      {cityWeather.map(({id, name, sys, weather, main, coord}) => {
```

you can do that in the constructor to make it cleaner